### ISSUE TITLE
issues rolling the latest dartdoc into google3

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/dartdoc#3562]
We're seeing issues rolling the current dartdoc into google3:

https://fusion2.corp.google.com/invocations/a94742f7-0a21-4518-b126-ad4f8a028292/targets/%2F%2Fdart%2Fdartdoc:dartdoc_e2e_test_run/tests

//dart/dartdoc:dartdoc_e2e_test_run

It looks like some golden test files might need to be regenerated:

```
Expected: '<!DOCTYPE html>\n'...
Expected: '<!DOCTYPE html>\n'
            '<html lang="en">\n'
            '<head>\n'
            '  <meta charset="utf-8">\n'
            '  <meta http-equiv="X-UA-Compatible" content="IE=edge">\n'
            '  <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1, user-
...
```

**Comment by devoncarew** [comment]
Ah, on closer inspection:

```
   Which: is different.
          Expected: ... rtdoc 7.0.1">\n  <me ...
            Actual: ... rtdoc 7.0.2">\n  <me ...
                                  ^
 ```

We may want to make that test not depend on the specific version text.

**Comment by srawlins** [comment]
> We may want to make that test not depend on the specific version text.

+1. I'd prefer to delete those golden tests in favor of unit tests.

**Comment by devoncarew** [comment]
fixed in association with b/321827375
